service/src/main/resources/db/changesets/20240923_update_input_defs_bucket_paths.yaml

```diff
@@ -0,0 +1,30 @@
+# Update input definitions bucket paths for service provided inputs
+databaseChangeLog:
+  - changeSet:
+      id: Update input definitions bucket paths for service provided inputs
+      author: mma
+      changes:
+        # update geneticMapsPath default_value from /plink-genetic-maps/GRCh38_fixed/
+        - update:
+            tableName: pipeline_input_definitions
+            where: id=3 AND name='geneticMapsPath'
```

Comment:
a slightly better approach would be to do something like `where pipeline_id=3` and probably the best approach would be `where pipeline_id = (select id from pipelines where name = "imputation_beagle")` or w/e the equivalent for liquidbase is.  No need for changes though